packages/instrumentation-pino/src/instrumentation.ts

```diff
@@ -67,10 +67,14 @@ export class PinoInstrumentation extends InstrumentationBase<PinoInstrumentation
           if (origMixin === undefined) {
             logger[mixinSym] = otelMixin;
           } else {
-            logger[mixinSym] = (ctx: object, level: number) => {
+            logger[mixinSym] = (
+              ctx: object,
+              level: number,
+              ...rest: unknown[]
+            ) => {
               return Object.assign(
                 otelMixin(ctx, level),
-                origMixin(ctx, level)
+                origMixin(ctx, level, ...rest)
```

Comment:
We could also explore an approach similar to https://github.com/DataDog/dd-trace-js/blob/213d56fb37bc7b929ee9cea99d21bb00eeafc412/packages/datadog-instrumentations/src/pino.js#L37-L51